front/pages/api/v1/w/[wId]/spaces/index.ts

```diff
@@ -65,7 +65,12 @@ async function handler(
 ): Promise<void> {
   switch (req.method) {
     case "GET":
-      const spaces = await SpaceResource.listWorkspaceSpaces(auth);
+      const allSpaces = await SpaceResource.listWorkspaceSpaces(auth);
```

Comment:
what about not returning it directly (or behind a flag)

Reply:
:+1: